cookiecutter/{{cookiecutter.project_name}}/CMakePresets.json

```diff
@@ -6,10 +6,16 @@
       "hidden": true,
       "generator": "Ninja",
       "binaryDir": "${sourceDir}/build/${presetName}",
+      "installDir": "${sourceDir}/stagedir",
```

Comment:
Now that the tests don't assume an `installDir` being set by the preset, we should not be mandating the `installDir` here